docs/fundamentals/setup-tooling.md

```diff
@@ -211,6 +214,18 @@ You add .NET Aspire components to your app like any other NuGet package using Vi
 
 For more information on .NET Aspire components, see [.NET Aspire components overview](components-overview.md).
 
+### Add a hosting component
```

Comment:
There was chatter that we didn't want to call these 'components' to confuse the term.  I personally think it's fine, but @davidfowl I think you wanted to call them "hosting resources"?  For p5 I don't think we have time to change the UI in VS (which calls it a hosting component).

This feels fine to me honestly though...but flagging.

Reply:
Yep! Resources

Reply:
Will the UI be updated then too? As of the latest VS preview internal bits, it reads as `.NET Aspire Hosting Component...`.